kurbo/src/point.rs

```diff
@@ -202,6 +202,30 @@ impl Point {
     pub fn is_nan(self) -> bool {
         self.x.is_nan() || self.y.is_nan()
     }
+
+    /// Get the member matching the given axis.
+    pub fn get(self, axis: Axis) -> f64 {
+        match axis {
+            Axis::Horizontal => self.x,
+            Axis::Vertical => self.y,
+        }
+    }
+
+    /// Get a mutable reference to the member matching the given axis.
+    pub fn get_mut(&mut self, axis: Axis) -> &mut f64 {
+        match axis {
+            Axis::Horizontal => &mut self.x,
+            Axis::Vertical => &mut self.y,
+        }
+    }
+
+    /// Set the member matching the given axis to the given value.
+    pub fn set(&mut self, axis: Axis, value: f64) {
+        match axis {
+            Axis::Horizontal => self.x = value,
+            Axis::Vertical => self.y = value,
+        }
+    }
```

Comment:
I don't much like the names of these methods; I think they're too unspecific/general for this functionality.

Reply:
I could call them e.g. `get_axis`, `set_axis`, etc? I do want short names, because in some algorithms we end up using them a lot.

Reply:
In other parts of Kurbo these are called "coordinate," or "coordinate value," as in "x-coordinate" and "y-coordinate". Perhaps something like `get_axis_coordinate` would be clearest, but a lot longer than your suggestions.